packages/@vue/cli/lib/util/features.js

```diff
@@ -3,9 +3,6 @@ const { toShortPluginId } = require('@vue/cli-shared-utils')
 
 exports.getFeatures = (preset) => {
   const features = []
-  if (preset.router) {
```

Comment:
Don't delete this line. It's for displaying preset features.
![image](https://user-images.githubusercontent.com/3277634/60728541-2ad26e00-9f73-11e9-9687-51d116a1780d.png)